### DISCUSSION TITLE
Is a tabline/statusLine planned

### DISCUSSION THREAD

**Discussion by giuseppeCruised** [JetBrains/ideavim#530] [Ideas]
Hi, im a big fan of Ideavim but i am really missing a beautiful statusline like airline in vim. Is anything of that sort planned? If not does anyone know if you can change the appearance of intellijs own (very small) statusline at the bottom of intellij, because it just doesnt look complete to me without a nice statusline. Thanks for the great work!

**Comment by AlexPl292** [comment]
Hi! We had some internal discussions about it, but not any plans. Sorry for that.
Here is an issue to track: https://youtrack.jetbrains.com/issue/VIM-1886/Airline-style-mini-status-bar-per-editor
